test/datatypes/datetime.coffee

```diff
@@ -1769,13 +1802,13 @@ describe 'DateTime.sameOrAfter', ->
     DateTime.parse('2000-02').sameOrAfter(DateTime.parse('2000-01-31T23:59:59.999')).should.be.true()
     DateTime.parse('2001').sameOrAfter(DateTime.parse('2000-12-31T23:59:59.999')).should.be.true()
 
-  it 'should accept cases where a has unknown values but is still deterministicly after or same as b', ->
-    DateTime.parse('2000-01-01T00:00:01').sameOrAfter(DateTime.parse('2000-01-01T00:00:01.0')).should.be.true()
-    DateTime.parse('2000-01-01T00:01').sameOrAfter(DateTime.parse('2000-01-01T00:01:00.0')).should.be.true()
-    DateTime.parse('2000-01-01T01').sameOrAfter(DateTime.parse('2000-01-01T01:00:00.0')).should.be.true()
-    DateTime.parse('2000-01-01').sameOrAfter(DateTime.parse('2000-01-01T00:00:00.0')).should.be.true()
-    DateTime.parse('2000-01').sameOrAfter(DateTime.parse('2000-01-01T00:00:00.0')).should.be.true()
-    DateTime.parse('2000').sameOrAfter(DateTime.parse('2000-01-01T00:00:00.0')).should.be.true()
+  it 'should handle cases where a has unknown values but is not deterministicly after or same as b', ->
```

Comment:
I won't comment on any more like this, but note that in all the cases where the test claimed `a` had a deterministic relationship to `b`, that really was (and still is) true.  The thing that changed is that the specification defined an approach to calculating these that is actually _imperfect_, thus the spec-compliant answer is less accurate than the _actually true_ answer.

The logic (and tests) need to follow the spec, but the test case descriptions shouldn't state something that's technically not true.